File: 프로그래머스/unrated/176963. 추억 점수/추억 점수.py
def solution(name, yearning, photo):
    info=dict(zip(name,yearning))
    result=[]
    for i in range (len(photo)):
        score=0
        for j in range (len(photo[i])):
            score+=info.get(photo[i][j],0)
        result.append(score)
    return result

File: 프로그래머스/1/176963. 추억 점수/추억 점수.js
function solution(name, yearning, photo) {
    var answer = [];
    for(let i=0; i<photo.length; i++){
        var sum=0
        for(let j=0; j<photo[i].length; j++){
            if(name.includes(photo[i][j])){
                var number = name.indexOf(photo[i][j])
                sum+=yearning[number]
            }
        }
        answer.push(sum)
    }
    return answer;
}